app/views/reservations/new.html.erb

```diff
@@ -4,6 +4,9 @@
   <%= simple_form_for([@game_world, @reservation]) do |f| %>
     <%= f.input :start_date, as: :date, html5: true %>
     <%= f.input :end_date, as: :date, html5: true %>
-    <%= f.submit %>
+    <%= f.submit%> <%= link_to "Go Back", game_world_path(@game_world), class: "btn btn-go-back" %>
   <% end %>
 </div>
+
+<div>
```

Comment:
I'm not sure, is this an empty div?

Reply:
okay just pushed again and fixed the empty div